roles/setup_pgd/tasks/validate_setup_pgd.yml

```diff
@@ -193,3 +193,9 @@
     init_data_query_res: null
     unix_stat: null
     pg_tde_query_result: null
+
+# Exclusive to pgd setup authentication
+- name: Start service oidentd, if not started
+  ansible.builtin.service:
+    name: oidentd
+    state: started
```

Comment:
Yeah, while this might help the immediate need, this probably needs to go somewhere more general.  We should continue discussing this specific issue in that other ticket.  (Sorry don't have reference offhand.)